src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html

```diff
@@ -267,19 +267,19 @@
 		<p>
 			Die Nutzung der App ist freiwillig. Es ist allein Ihre Entscheidung, ob Sie
 			die App installieren, welche App-Funktionen Sie nutzen und ob Sie Daten mit
-			anderen teilen. Alle App-Funktionen, die eine Datenweitergabe Ihrer Begegnungs- 
-			und Gesundheitsdaten erfordern, holen vorher Ihre ausdrückliche Einwilligung ein. 
-			Falls Sie eine Einwilligung nicht erteilen oder nachträglich widerrufen, entstehen Ihnen
+			anderen teilen. Alle App-Funktionen, die eine Weitergabe Ihrer Begegnungs- 
+			oder Gesundheitsdaten erfordern, holen vorher Ihr ausdrückliches Verständnis ein. 
```

Comment:
line 271: "Verständnis" - should this not be "Einverständnis"?

Reply:
Maybe:
> Bei allen App-Funktionen, die eine Weitergabe Ihrer Gesundheitsdaten erfordern, werden Sie vorab aufgefordert, zuzustimmen.

Reply:
Eagle Eye 👍  – fixed with d5f54c5